Gemfile

```diff
@@ -0,0 +1,28 @@
+source 'https://rubygems.org'
+
+ruby '3.3.1'
+
+gem 'bootsnap', require: false
+gem 'dotenv-rails', groups: %i[development test]
+gem 'importmap-rails'
+gem 'jbuilder'
+gem 'pg', '~> 1.1'
+gem 'puma', '>= 5.0'
+gem 'rails', '~> 7.1.3', '>= 7.1.3.4'
+gem 'sprockets-rails'
+gem 'stimulus-rails'
+gem 'turbo-rails'
+gem 'tzinfo-data', platforms: %i[windows jruby]
+
+group :development, :test do
+  gem 'rspec-rails'
+end
+
+group :development do
+  gem 'web-console'
+end
+
+group :test do
+  gem 'capybara'
+  gem 'selenium-webdriver'
```

Comment:
Why do we need this?

Reply:
@SKH-WTAG bhai, we need `capybara` to write test files that helps to simulate how a user would interact with web app. It provides domain-specific-language for writing tests. The `selenium-webdriver` is a automating tool for web browser. It allows `capybara` to run tests in real browser environment.

Reply:
For now, we won't test views using browser. We can add this later if needed.